src/tests/ut/visit/visit_test.cpp

```diff
@@ -2,47 +2,63 @@
 
 #include "visit_test.hpp"
 
+#include <algorithm>
+
 namespace
 {
-TEST_F(AssociationTest, ExtentDemo)
+TEST_F(VisitTestsFixture, ExtentDemo)
 {
     auto content = Visit::getExtent().size();
     auto expected = 3;
     EXPECT_EQ(content, expected);
 }
 
-TEST_F(AssociationTest, AssociationDemo)
+TEST_F(VisitTestsFixture, AssociationDemo)
 {
     auto content = visit1->getDoctorAssociation()->getLastName();
     auto expected = "Tracz";
     EXPECT_EQ(content, expected);
 }
 
-TEST_F(AssociationTest, AssociationDemo2)
+TEST_F(VisitTestsFixture, AssociationDemo2)
 {
     auto content = doc2->getVisitAssociations().begin()->get()->getVisitInformation();
     auto expected = "Tworze klub ninja!";
     EXPECT_EQ(content, expected);
 }
 
-TEST_F(AssociationTest, AssociationDemo3)
+TEST_F(VisitTestsFixture, AssociationDemo3)
 {
     std::string content;
     auto associations = doc2->getVisitAssociations();
```

Comment:
As you already change something within those tests, please consider switching auto to const auto where possible.

Reply:
Replaced auto with const auto& wherever possible (or not missed :D)

Reply:
For "things" that are created on-fly such as:
`const auto& expected = 3;`
we shouldn't use & as there might be an issue with gtest macro resolving (if I remember correctly) 
Would be good to adjust that to `const auto` and the other places, where we just copy values would be fine too imo

Reply:
You are completely right. I got ahead of myself and didn't even check those all are copies. Thanks for spotting that!